machines/bleShare/scan/scanMachine.ts

```diff
@@ -137,6 +139,26 @@ export const scanMachine =
         inactive: {
           entry: 'removeLoggers',
         },
+        checkStorage: {
+          invoke: {
+            src: () =>
+              Promise.resolve(
+                Storage.isMinimumLimitReached(
+                  'minimumStorageRequiredForAuditEntryInMB'
+                )
+              ),
+            onDone: [
+              {
+                cond: (_context, event) => event.data === true,
```

Comment:
same here as well. can we extract into a function.

Reply:
Refactored